spec/features/campaign_requests_spec.rb

```diff
@@ -55,7 +55,7 @@
 Some comment"})
 
     user_makes_a_campaign_request(
-      campaign_title: "Workplace pensions",
+      title: "Workplace pensions",
```

Comment:
Looks like the changes in this file can be squashed into the previous commit as they're to do with the updated campaign form.